lib/systems/src/TorqueControllers.cpp

```diff
@@ -159,15 +159,31 @@ void TorqueControllerLoadCellVectoring::tick(
 void TorqueControllerPIDTV::tick(const SysTick_s &tick, const PedalsSystemData_s &pedalsData, float vx_b, float wheel_angle_rad, float yaw_rate)
 {
 
-    pid_input_.Vx_B = vx_b;
-    pid_input_.WheelDeltarad = wheel_angle_rad;
-    pid_input_.YawRaterads = yaw_rate;
-    pid_input_.FR_in = 1.0;
-    pid_input_.RR_in = 1.0;
-    pid_input_.FL_in = 1.0;
-    pid_input_.RL_in = 1.0;
-
-    tv_pid_.step();
-    const PID_TV::ExtY_PID_TV_T& out = tv_pid_.getExternalOutputs();
-    
+    float accelRequest = pedalsData.accelPercent;
+    auto torqueRequest = accelRequest * AMK_MAX_TORQUE;
+
+    if (tick.triggers.trigger100)
+    {
+        pid_input_.Vx_B = vx_b;
```

Comment:
Uh, does this work?

You pass a pointer to the ExtY_PID_TV_T struct when you construct the PIDTV controller
```c++
tv_pid_.initialize();
        tv_pid_.setExternalInputs(&pid_input_);
        pid_input_.PID_P = 2.0;
        pid_input_.PID_I = 1.0;
        pid_input_.PID_D = 0.0;
        pid_input_.PID_N = 100;
``` 

But you then proceed to dereference the pointer in PID_TV
```c++
void setExternalInputs(const ExtU_PID_TV_T *pExtU_PID_TV_T)
  {
    PID_TV_U = *pExtU_PID_TV_T;
  }
``` 

so it is just operating on a copy of the data unless I am missing something.